### ISSUE TITLE
No `avocado-plugins-vt` packet on Fedora 27

### ISSUE THREAD

**Issue by abitrolly** [avocado-framework/avocado-vt#1357]
http://avocado-vt.readthedocs.io/en/latest/GetStartedGuide.html#fedora-and-enterprise-linux

```
✗ sudo yum install avocado-plugins-vt
Last metadata expiration check: 0:58:31 ago on Fri 23 Feb 2018 12:10:03 PM +03.
No match for argument: avocado-plugins-vt
Error: Unable to find a match
```

**Comment by clebergnu** [comment]
@abitrolly have you installed the Avocado repository? Documentation is here:

http://avocado-framework.readthedocs.io/en/60.0/GetStartedGuide.html#fedora-from-avocado-s-own-repo

**Comment by abitrolly** [comment]
No. I think no. I probably tried to install it with `pip`. Maybe Avocado docs should be updated to mention that Avocado VT can only be used and installed if Avocado itself was installed from custom repo. Mention that repo explicitly could help.

**Comment by clebergnu** [comment]
Hi @KevinWhalen ,

Avocado-VT is on a different module, named `avocado-vt`.  But that is indeed missing in the documentation.  Thanks for spotting that.
